checkmate/templates/admin/allow_rule.html.jinja2

```diff
@@ -0,0 +1,26 @@
+{% extends "checkmate:templates/admin/base.html.jinja2" %}
+{% block content %}
+
+  <h2>Add to Allow List</h2>
+
+  {% if messages %}
+    <div>
+      {% for message in messages %}
+        <p>{{ message.detail }}</p>
+      {% endfor %}
+    </div>
+  {% endif %}
+
+  {% if allow_rule %}
+    <p>Allowed as {{ allow_rule.rule }} with hash {{ allow_rule.hash }}</p>
```

Comment:
Sticking to the same format we had in [Checkmate.allow_url](https://github.com/hypothesis/checkmate/blob/ca0a9201bded2e092b55f4a0f0918f5dc10dff29/bin/add_to_allow_list.py#L111).